variants/classical/classical.go

```diff
@@ -51,7 +51,7 @@ func Blank(phase dip.Phase) *state.State {
 }
 
 func Start() (result *state.State, err error) {
-	result = state.New(start.Graph(), &phase{1901, cla.Spring, cla.Movement}, BackupRule)
+	result = state.New(start.Graph(), &phase{1901, cla.Spring, cla.Movement, orders.ClassicalParser}, BackupRule)
```

Comment:
Actually, the classical/orders.ClassicalParser ought to be called just Parser as well, since it's already in the classical package... But that's probably for another PR.